inst/documents/dummy.qmd

```diff
@@ -62,9 +62,10 @@ knitr::spin_child(params$script)
 #| include: false
 
 python_packages <- NULL
-# if (file.exists(file.path(params$tmpdir, "python_imports.json"))){
-#   python_packages <- whirl:::python_package_info(file.path(params$tmpdir, "python_imports.json"))
-# }
+
+if (file.exists(file.path(params$tmpdir, "python_imports.json"))){
+  python_packages <- whirl:::python_package_info(file.path(params$tmpdir, "python_imports.json"))
```

Comment:
may be astupid question but do we have test our function with venv ? and what happens is our users use something else then pip ? such as poetry or uv (https://docs.astral.sh/uv/) ?

Reply:
I have not, and not really sure what would happen. @Michaelsvingel do you know?

Reply:
Talked with Michael, and he says it should work. We should test it more in the future, and have also opened an issue #127 to create some better tests for it.